release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -814,6 +840,11 @@ In the table, features are marked with the following statuses:
 |GA
 |DEP
 
+|Use of `v1` in `apiVersion`
```

Comment:
Table entry for https://github.com/openshift/openshift-docs/issues/26801#issuecomment-758051819
@soltysh

Reply:
That's not quite right, this only apply to openshift resources, iow. image streams, deployment configs, build configs, routes, etc. I'm not sure if we have somewhere a list of openshift-specific resources which we could point to.

Reply:
Probably worth pointing to https://docs.openshift.com/container-platform/4.6/rest_api/index.html and stating that every resource that has `*.openshift.io` in `apiVersion` requires that to match that value from that page and can't be just `v1`. Does that make sense?

Reply:
That's a good clarification! I'll make some edits accordingly. Thanks :)

Reply:
Updated now